### ISSUE TITLE
Exception with Python 2.6

### ISSUE THREAD

**Issue by aldemira** [simple-salesforce/simple-salesforce#44]
While using this module with Python 2.6 I'm getting the following exception:
 File "/usr/lib64/python2.6/site-packages/requests/models.py", line 776, in json
    return json.loads(self.text, **kwargs)
  File "/usr/lib64/python2.6/json/__init__.py", line 318, in loads
    return cls(encoding=encoding, **kw).decode(s)
TypeError: __init__() got an unexpected keyword argument 'object_pairs_hook'

the solution is to use simplejson 
And change api.py import to 
import simplejson as json

**Comment by dylanjbarth** [comment]
I ran into this error as well 

```python
.../python2.7/site-packages/simple_salesforce/api.pyc in create(self, data)
    467         result = self._call_salesforce('POST', self.base_url,
    468                                        data=json.dumps(data))
--> 469         return result.json(object_pairs_hook=OrderedDict)
    470 
    471     def upsert(self, record_id, data, raw_response=False):

TypeError: json() got an unexpected keyword argument 'object_pairs_hook'
```
Turns out my venv had a very old version of requests, upgrading to `requests==2.7.0` resolved this issue.

**Comment by demianbrecht** [comment]
Closing as this seem to be resolved by `requests` (2.6 unit tests are now passing in travis too).
